src/sagemaker_xgboost_container/serving_mms.py

```diff
@@ -77,18 +82,21 @@ def _set_mms_configs(is_multi_model, handler):
     # Max heap size = (max workers + max job queue size) * max payload size * 1.2 (20% buffer) + 128 (base amount)
     max_heap_size = ceil((max_workers + max_job_queue_size) * (int(max_content_length) / 1024 ** 2) * 1.2) + 128
 
-    os.environ["SAGEMAKER_NUM_MODEL_WORKERS"] = '1'
-    os.environ["SAGEMAKER_MODEL_JOB_QUEUE_SIZE"] = '2'
     os.environ["SAGEMAKER_MMS_MODEL_STORE"] = '/'
     os.environ["SAGEMAKER_MMS_LOAD_MODELS"] = ''
+    os.environ["SAGEMAKER_MMS_DEFAULT_HANDLER"] = handler
 
-    if not os.getenv("SAGEMAKER_BIND_TO_PORT", None):
-        os.environ["SAGEMAKER_BIND_TO_PORT"] = str(PORT)
+    # Users can define port
+    _set_default_if_not_exist("SAGEMAKER_BIND_TO_PORT", str(PORT))
 
-    os.environ["SAGEMAKER_MAX_HEAP_SIZE"] = str(max_heap_size) + 'm'
-    os.environ["SAGEMAKER_MAX_DIRECT_MEMORY_SIZE"] = os.environ["SAGEMAKER_MAX_HEAP_SIZE"]
-    os.environ["SAGEMAKER_MAX_REQUEST_SIZE"] = str(max_content_length)
-    os.environ["SAGEMAKER_MMS_DEFAULT_HANDLER"] = handler
+    # Multi Model Server configs, exposed to users as env vars
+    _set_default_if_not_exist("SAGEMAKER_NUM_MODEL_WORKERS", 1)
```

Comment:
Could we please declare default consts in caps, at the top of the file, like how we have declared `PORT`?

Reply:
Sure, I can make that change.